src/olympia/versions/models.py

```diff
@@ -581,6 +581,21 @@ def unreviewed_files(self):
         """A File is unreviewed if its status is amo.STATUS_AWAITING_REVIEW."""
         return self.files.filter(status=amo.STATUS_AWAITING_REVIEW)
 
+    @property
+    def is_ready_for_auto_approval(self):
+        """Return whether or not this version could be *considered* for
+        auto-approval.
+
+        Does not necessarily mean that it would be auto-approved, just that it
+        passes the most basic criteria to be considered a candidate by the
+        auto_approve command."""
+        return (
+            self.addon.status == amo.STATUS_PUBLIC and
+            self.addon.type == amo.ADDON_EXTENSION and
+            self.is_webextension and
+            self.is_unreviewed and
```

Comment:
This will have the weird (imo) self-effect of saying every approved version, whether auto-approved or not, isn't ready for auto-approval.  Which is technically correct as you can't approve it twice.  But unexpected (again, imo),